custom/icds_reports/static/js/angular-services/baseControllers.service.js

```diff
@@ -74,7 +74,10 @@ window.angular.module('icdsApp').factory('baseControllersService', function() {
                     vm.steps['map'].label = 'Map View: ' + locType;
                 }
             };
-            vm.loadDataFromResponse = function(usePercentage, forceYAxisFromZero) {
+            vm.loadDataFromResponse = function(usePercentage, forceYAxisFromZero, overrideStep) {
```

Comment:
mobile dashboard needs to display some map data in the "chart" step, and this was hardcoded to use the `vm.step` from the controller which made it impossible to do without this change.